portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +31,22 @@
 /** Servlet that returns some example content and handles data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> commentsStorage = new ArrayList<>();
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private int max_comments_num = 5;
```

Comment:
I think you could make this a constant, i.e.,

static final int MAX_COMMENTS_NUM = 5;

Reply:
Good suggestion on that! While I have already changed the constant as a variable in the latest update. So, the number can be inserted and changes with the viewer's choice.